k8s-readwrite.tf

```diff
@@ -226,6 +222,15 @@ resource "kubernetes_deployment_v1" "apiary_hms_readwrite" {
             name  = "LIMIT_PARTITION_REQUEST_NUMBER"
             value = var.hms_rw_request_partition_limit == "" ? "" : var.hms_rw_request_partition_limit
           }
+          env {
+            name  = "DATANUCLEUS_CONNECTION_POOLING_TYPE"
+            value = var.hms_rw_datanucleus_connection_pooling_type
+          }
+          env {
```

Comment:
Why do we have `DATANUCLEUS_CONNECTION_POOL_MAX_POOLSIZE` setting but at the same time we add dynamically connection pool configuration? should not be this part of the dynamic configuration?

Reply:
I thought about it a lot. It can be but it should not be. It is way too important and can be missed. We need to add an explicit setting for the pool size.

Reply:
this setting by the way is shared across all pool types as well, so it's safe to make it explicit.